internal/backup/pvc_action.go

```diff
@@ -122,7 +123,7 @@ func (p *PVCBackupItemAction) Execute(item runtime.Unstructured, backup *velerov
 			GenerateName: "velero-" + pvc.Name + "-",
 			Namespace:    pvc.Namespace,
 			Labels: map[string]string{
-				velerov1api.BackupNameLabel: backup.Name,
+				velerov1api.BackupNameLabel: label.GetValidName(backup.Name),
 			},
 		},
 		Spec: snapshotv1beta1api.VolumeSnapshotSpec{
```

Comment:
Lines 144 and 145 need it, too.

This is tricky because we'll be sanitizing names that we rely on being accurate in the API server :(

Reply:
Those get addressed in the `util.AddLabels` function